x11-misc/bbkeys/bbkeys-0.9.1-r1.ebuild

```diff
@@ -16,10 +18,10 @@ DEPEND="${RDEPEND}
 	virtual/pkgconfig"
 
 src_install() {
-	emake DESTDIR="${D}" install || die
-	rm -rf "${D}"/usr/share/doc
-	dodoc AUTHORS BUGS ChangeLog NEWS README || die
+	emake DESTDIR="${ED%/}" install
+	rm -rf "${ED%/}"/usr/share/doc || die
+	einstalldocs
 
-	echo PRELINK_PATH_MASK=/usr/bin/bbkeys > "${T}"/99bbkeys
-	doenvd "${T}"/99bbkeys || die
+	echo PRELINK_PATH_MASK=/usr/bin/bbkeys > "${T}"/99bbkeys || die
```

Comment:
`echo PRELINK_PATH_MASK=\"${EPREFIX}/usr/bin/bbkeys\" > "${T}"/99bbkeys || die`
check that the double quotes remain in the file